prime-router/metadata/schemas/covid-19.schema

```diff
@@ -228,6 +229,7 @@ elements:
 
   - name: ordering_facility_email
     type: EMAIL
+    canBeBlank: true
     hl7Field: ORC-23-4
```

Comment:
Another new field indicates that blank is a valid value for this field. Suppresses warnings and errors if this field is true.

Reply:
Remove canBeBlank as a field and add a couple of types where it is necessary. Here's logic table which I put in code. 

```
| Type         | Cardinality | no column on input                   | empty input value                    | Invalid input value | valid input value |
|--------------|-------------|--------------------------------------|--------------------------------------|---------------------|-------------------|
| FOO          | 0..1        | mapper -> default -> empty + warning | mapper -> default -> empty + warning | empty + warning     | value             |
| FOO          | 1..1        | mapper -> default -> error           | mapper -> default -> error           | error               | value             |
| FOO_OR_BLANK | 0..1        | mapper -> default -> empty + warning | empty                                | empty + warning     | value             |
| FOO_OR_BLANK | 1..1        | mapper -> default -> error           | empty                                | error               | value             |

```

Reply:
I think there's still room for further simplification.  note that
`FOO 0..1`   and 'FOO_OR_BLANK 0..1`   are nearly identical - the minor difference that one gives a Warning on empty input values, and the other is silent.